content/blog/2010/2010-03-02-what-new-features-do-you-want-to-see.md

```diff
@@ -11,7 +11,7 @@
 - jenkinsci
 :author: rtyler
 ---
-Michael Donohue, a Hudson developer who has taken on the role of master bug triage guy for Hudson, does something regularly which I've really come to appreciate as a Hudson developer myself: he sends out emails to the dev list with the top 10 voted issues at that time. This gives those of us in the Hudson development community a good sense of what's really important to our users, which in turn helps us decide where to focus our efforts. If you're interested, you can see <a href="https://issues.hudson-ci.org/secure/IssueNavigator.jspa?mode=hide&requestId=10045">the top voted issues</a> over at <a href="https://issues.hudson-ci.org">our JIRA server.</a>
+Michael Donohue, a Hudson developer who has taken on the role of controller bug triage guy for Hudson, does something regularly which I've really come to appreciate as a Hudson developer myself: he sends out emails to the dev list with the top 10 voted issues at that time. This gives those of us in the Hudson development community a good sense of what's really important to our users, which in turn helps us decide where to focus our efforts. If you're interested, you can see <a href="https://issues.hudson-ci.org/secure/IssueNavigator.jspa?mode=hide&requestId=10045">the top voted issues</a> over at <a href="https://issues.hudson-ci.org">our JIRA server.</a>
```

Comment:
Not sure if controller applies in this case, I think this is referring to a person as a "master" of something. This may be a case where we need a different term?